web/src/components/OpenEndedInputTag/OpenEndedInputTag.tsx

```diff
@@ -0,0 +1,167 @@
+import {
+  ChangeEventHandler,
+  ForwardedRef,
+  SyntheticEvent,
+  useCallback,
+} from 'react'
+
+import { Input, InputProps } from '@chakra-ui/input'
+import {
+  Circle,
+  Stack,
+  Wrap,
+  WrapItem,
+  WrapItemProps,
+  WrapProps,
+  Text,
+} from '@chakra-ui/layout'
+import { TagCloseButtonProps, TagLabelProps, TagProps } from '@chakra-ui/tag'
+
+import TagInput from '../TagInput/TagInput'
+
+import { maybeCall, MaybeFunc } from './../../maybe'
+
+type MaybeIsInputProps<P> = MaybeFunc<[isInput: boolean, index?: number], P>
+type MaybeTagProps<P> = MaybeFunc<[tag: string, index?: number], P>
+
+export type OpenEndedInputTagProps = InputProps & {
+  tags?: string[]
+  onTagsChange?: (event: SyntheticEvent, tags: string[]) => void
+  onTagAdd?: (event: SyntheticEvent, value: string) => void
+  onTagRemove?: (event: SyntheticEvent, index: number) => void
+
+  addKeys?: string[]
+  question?: string
+
+  onChange: ChangeEventHandler<HTMLInputElement>
+
+  placeholder: string
+  value: string
+
+  wrapProps?: WrapProps
+  wrapItemProps?: MaybeIsInputProps<WrapItemProps>
+  tagProps?: MaybeTagProps<TagProps>
+  tagLabelProps?: MaybeTagProps<TagLabelProps>
+  tagCloseButtonProps?: MaybeTagProps<TagCloseButtonProps>
+}
+
+const OpenEndedInputTag = (
+  {
+    tags = [],
+    onTagsChange,
+    onTagAdd,
+    onTagRemove,
+    addKeys = ['Enter', ' '],
+    question,
+    onChange,
+    placeholder,
+    value,
+    wrapProps,
+    wrapItemProps,
+    tagProps,
+    tagLabelProps,
+    tagCloseButtonProps,
+    ...props
+  }: OpenEndedInputTagProps,
+  ref: ForwardedRef<HTMLInputElement>
+) => {
+  const addTag = useCallback(
+    (event: SyntheticEvent, tag: string) => {
+      onTagAdd?.(event, tag)
+      if (event.isDefaultPrevented()) return
+
+      onTagsChange?.(event, tags.concat([tag]))
+    },
+    [tags, onTagsChange, onTagAdd]
+  )
+
+  const removeTag = useCallback(
+    (event: SyntheticEvent, index: number) => {
+      onTagRemove?.(event, index)
+      if (event.isDefaultPrevented()) return
+      onTagsChange?.(event, [...tags.slice(0, index), ...tags.slice(index + 1)])
+    },
+    [tags, onTagsChange, onTagRemove]
+  )
+
+  const handleRemoveTag = useCallback(
+    (index: number) => (event: SyntheticEvent) => {
+      removeTag(event, index)
+    },
+    [removeTag]
+  )
+
+  const onKeyDown = props.onKeyDown
+  const handleKeyDown = useCallback(
+    (event: KeyboardEvent<HTMLInputElement>) => {
+      onKeyDown?.(event)
+
+      if (event.isDefaultPrevented()) return
+      if (event.isPropagationStopped()) return
+
+      const { currentTarget, key } = event
+      const { selectionStart, selectionEnd } = currentTarget
+
+      if (addKeys.indexOf(key) > -1 && currentTarget.value) {
+        addTag(event, currentTarget.value)
+        if (!event.isDefaultPrevented()) {
+          currentTarget.value = ''
```

Comment:
Your Field is not cleared (most likely because this has no effect on the redwood form. 
<img width="817" alt="Capture d’écran 2023-06-05 à 14 44 00" src="https://github.com/x-image-privacy/privacyAdvisor/assets/39373170/9a3dc491-7471-44df-96c3-0a0022720b56">
You can simply call `onChange("")` on the following line to propagate the value change to the form component.

Reply:
Yes, it works!